manticore/core/manticore.py

```diff
@@ -39,7 +39,7 @@ class ManticoreBase(Eventful):
     :ivar dict context: Global context for arbitrary data storage
     '''
 
-    _published_events = {'start_run', 'finish_run'}
+    _published_events = {'start_run', 'finish_run', 'generate_testcase'}
```

Comment:
Btw. adding `generate_testcase` here prevents from getting a warning when we publish a `will_generate_testcase` event.

I am not sure if it is good that the prefix is ommitted - as this probably means we can also freely publish `did_generate_testcase` which is not there anywhere...